src/raftstore/store/keys.rs

```diff
@@ -57,30 +57,29 @@ pub const SNAPSHOT_RAFT_STATE_SUFFIX: u8 = 0x04;
 // For region meta
 pub const REGION_STATE_SUFFIX: u8 = 0x01;
 
-pub fn store_ident_key() -> Vec<u8> {
-    STORE_IDENT_KEY.to_vec()
-}
-
-pub fn prepare_bootstrap_key() -> Vec<u8> {
-    PREPARE_BOOTSTRAP_KEY.to_vec()
+#[inline]
+fn make_region_prefix(region_id: u64, suffix: u8) -> [u8; 11] {
+    let mut key = [0; 11];
```

Comment:
can we define a const var for 11 or other const sizes?

Reply:
I think using 11 is more readable here. Because following code will take slice with magic number, use a const will make reader doubt if the key is long enough.